.github/workflows/version.yml

```diff
@@ -1,8 +1,6 @@
 name: Create a new version
 
-on:
-    push:
-        branches: [master]
+on: push
```

Comment:
So we want to generate a new tag for each push to any branch? It seems like we will have a huge amount of tags in no time in that case

Reply:
ooops, this is another change I forgot to revert after testing